src/server/requests/serverrequests.cpp

```diff
@@ -1490,14 +1490,18 @@ ExitCode ServerRequests::addSync(int driveDbId, const QString &localFolderPath,
     Sync sync;
     sync.setDbId(syncDbId);
     sync.setDriveDbId(driveDbId);
-    sync.setLocalPath(QStr2Path(localFolderPath));
+    SyncName localPath = Str2SyncName(localFolderPath.toStdString());
```

Comment:
```suggestion
    SyncName localPath = QStr2Path(localFolderPath);
```
To keep a path type for a variable that stores a path.

Reply:
Done in 74f95fc7778e687ed0be1733f5c8ae3bf5b60b73